Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.7
+
+RUN apt -y update
+RUN apt -y install git
+RUN git clone https://github.com/jhuapl-boss/ingest-client
+RUN pip install ./ingest-client
```

Comment:
If you install directly from Github then you don't incur an extra layer that is not needed.

`RUN pip install https://github.com/jhuapl-boss/ingest-client`

Reply:
Got it! Will fix this in the next commit. 

Original rationale was that I wanted to have an opportunity for users to mount custom plugin files; but don't think that makes sense for the base image here.